apps/common_logging_processors.go

```diff
@@ -88,3 +88,63 @@ func genericAccessLogParser(ctx context.Context, processorType, tag, uid string)
 	c = append(c, mf.Components(ctx, tag, uid)...)
 	return c
 }
+
+func genericAccessLogParserV2(ctx context.Context, processorType, tag, uid string) []confgenerator.InternalLoggingProcessor {
```

Comment:
The `genericAccessLogParserV2` naming maybe temporary during this migration, but i think it would be better to have a more descriptive naming. 

My suggestion is `genericAccessLogParserAsInternalLoggingProcessor` to describe the return type is `InternalLoggingProcessor`.

Also, let's add a `TODO:` to update the naming when the `old` version is cleaned up.